k8s/namespaces/bsp/bulk-scan-payment-processor/aat.yaml

```diff
@@ -25,7 +25,7 @@ spec:
               idam-users-bulkscan-username: IDAM_USERS_BULKSCAN_USERNAME
               idam-users-bulkscan-password: IDAM_USERS_BULKSCAN_PASSWORD
               idam-client-secret: IDAM_CLIENT_SECRET
-              payments-queue-send-connection-string: PAYMENTS_QUEUE_WRITE_CONNECTION_STRING
+              payments-queue-send-shared-access-key: PAYMENTS_QUEUE_WRITE_ACCESS_KEY
```

Comment:
already map in values?

Reply:
that's test config. charts already have it for app itself